ql/ql/src/codeql_ql/ast/Yaml.qll

```diff
@@ -64,4 +64,15 @@ class QlRefDocument extends YamlDocument {
       value = n.lookup("postprocess").(YamlSequence).getElement(_)
     )
   }
+
+  predicate isPrintAst() {
```

Comment:
A more ambitious solution would be to resolve the the query pointed to by the `.qlref` file, and then only flag cases where the query has kind `problem` or `path-problem`, but we can leave that for a future improvement.